ldes-server-port-fetch-rest/src/main/java/be/vlaanderen/informatievlaanderen/ldes/server/rest/caching/EtagCachingStrategy.java

```diff
@@ -20,24 +20,19 @@ public EtagCachingStrategy(final AppConfig appConfig) {
 	}
 
 	@Override
-	public String generateCacheIdentifier(EventStream eventStream) {
-		return DigestUtils.sha256Hex(appConfig.getHostName() + "/" + eventStream.collection());
-	}
-
-	@Override
-	public String generateCacheIdentifier(String collectionName) {
-		return DigestUtils.sha256Hex(appConfig.getHostName() + "/" + collectionName);
+	public String generateCacheIdentifier(String collectionName, String language) {
+		return sha256Hex(appConfig.getHostName() + "/" + collectionName + "?lang=" + language);
```

Comment:
the "?lang=" is something that is added by the Controllers or is that a param that explicitly need to be added?